src/shared/varlink-io.systemd.Machine.c

```diff
@@ -38,6 +38,15 @@ static SD_VARLINK_DEFINE_METHOD(
                 SD_VARLINK_FIELD_COMMENT("The name of a machine to terminate."),
                 SD_VARLINK_DEFINE_INPUT(name, SD_VARLINK_STRING, 0));
 
+static SD_VARLINK_DEFINE_METHOD(
+                Kill,
+                SD_VARLINK_FIELD_COMMENT("The name of a machine to send signal to."),
+                SD_VARLINK_DEFINE_INPUT(name, SD_VARLINK_STRING, 0),
+                SD_VARLINK_FIELD_COMMENT("Identifier that specifies what precisely to send the signal to (either 'leader' or 'all')."),
+                SD_VARLINK_DEFINE_INPUT(whom, SD_VARLINK_STRING, SD_VARLINK_NULLABLE),
+                SD_VARLINK_FIELD_COMMENT("Numeric UNIX signal integer."),
+                SD_VARLINK_DEFINE_INPUT(signal, SD_VARLINK_INT, 0));
```

Comment:
as with terminate/unregister, the polkit field and the pid thing are missing, and name needs to be nullable